tests/DI/PhpReflection.getParameterType.phpt

```diff
@@ -27,3 +27,4 @@ Assert::same('Test\B', PhpReflection::getParameterType($params[1]));
 Assert::same('array', PhpReflection::getParameterType($params[2]));
 Assert::same('callable', PhpReflection::getParameterType($params[3]));
 Assert::null(PhpReflection::getParameterType($params[4]));
+Assert::same('Test\B', PhpReflection::getParameterType($params[5]));
```

Comment:
You should add `tests/DI/PhpReflection.getParameterType.php71.phpt` (with `@phpversion >= 7.1` annotation) with real nullable types.